add-slb-nncp.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: nmstate.io/v1                                                       
+kind: NodeNetworkConfigurationPolicy                                            
+metadata:                                                                       
+  name: slb
+spec:
+  capture:
+    primary-nic: interfaces.description == "primary"
+    secondary-nic: interfaces.description == "secondary"
+  desiredState:
+    interfaces:
+    - name: brcnv-iface
+      type: ovs-interface
+      state: up
+      mac-address: "{{ capture.primary-nic.interfaces.0.mac-address }}"
+      ipv4:
+        enabled: true
+        dhcp: true
```

Comment:
Should we be explicit with

```
      auto-dns: true
      auto-gateway: true
      auto-routes: true
```

?

Reply:
Those flags a true by default if we set `dhcp: true`

Reply:
I see